server/store/views.py

```diff
@@ -93,3 +99,50 @@ def list(self, request, formatting=None):
         active = SiteConfig.objects.get(active=True)
         serializer = SiteConfigSerializer(active)
         return Response(serializer.data)
+
+
+@require_http_methods(["POST"])
```

Comment:
This is known! The status 400, csrf errors. Still using the `@csrf_exempt` decorator locally as you directed to bypass now. Would this be something that won't crop up when deployed?

<img width="856" alt="Screenshot 2023-03-24 at 12 58 25 PM" src="https://user-images.githubusercontent.com/1331216/227628231-b292584a-fc4e-494e-be8a-8e3bdd8031c2.png">

Reply:
I was only using this to work around #97 when developing locally. It's not something that should be needed otherwise.